api/enterprise-api-ref/first-time-setup.markdown

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Fist Time Setup API
+published: true
+---
+
+The First Time Setup API enables creation of the initial administrator user. This API is only accessible when the system hasn't been previously configured. Once setup is complete, this API becomes permanently inaccessible.
+
+## Get setup code
+
+The setup code is generated during hub installation. If you need a new setup code, execute this command on your hub:
+
+```console
+sudo cf-hub --new-setup-code
+```
+
+**Note:** The setup code will expire after 1 hour.
+
+## Check setup status
+
+First, check if the system has already been set up. This endpoint returns the current setup status of the system.
+
+**URI:** https://hub.example.com/api/setup/status
+
+**Method:** GET
+
+**Example request (curl):**
+
+```console
+curl -X GET \
```

Comment:
does this not require any authentication then? Would make sense... just checking.

Reply:
No, this doesn't require